hal/ia32/pci.c

```diff
@@ -145,12 +129,12 @@ int hal_pciGetDevice(pci_id_t *id, pci_dev_t *dev, void *caps)
 				if ((id->device != PCI_ANY) && (id->device != (dv >> 16)))
 					continue;
 
-				cl = hal_pciGet(b, d, f, 2) >> 16;
+				cl = _hal_pciGet(b, d, f, 2) >> 16;
 
 				if ((id->cl != PCI_ANY) && (id->cl != cl))
 					continue;
 
-				val = hal_pciGet(b, d, f, 0xb);
+				val = _hal_pciGet(b, d, f, 0xb);
 
 				if ((id->subdevice != PCI_ANY) && (id->subdevice != ((val >> 16) & 0xffff)))
```

Comment:
The mask is probably unnecessary.
val is `unsigned int`
after shift right by 16 it will be `0000xxxx`.

Reply:
Fixed in next commit, thanks.